python/cudf/cudf/_lib/utils.pyx

```diff
@@ -44,8 +44,13 @@ cdef table_to_dataframe(cudf_table* c_table, int_col_names=False):
     df = DataFrame()
     for i in range(c_table[0].num_columns()):
         c_col = c_table[0].get_column(i)
-        col = gdf_column_to_column(c_col, int_col_names)
-        df.insert(i, col.name, col)
+        col = gdf_column_to_column(c_col)
+        name = None
+        if c_col.col_name is not NULL:
```

Comment:
```suggestion
        if c_col.col_name != NULL:
```

Reply:
`is not NULL` seems to be used in the Cython docs:

https://cython.readthedocs.io/en/latest/src/userguide/extension_types.html

Reply:
Ah it seems some code was using `!= NULL` and some was using `is not NULL`. Seemed clearer to prefer the former, but no strong feelings about it.

Reply:
Marking resolved